compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -554,7 +554,7 @@ expandSdkPackages logger lfVersion dars = do
     expand mbSdkPath fp
       | fp `elem` basePackages = pure fp
       | isSdkPackage fp = case mbSdkPath of
-            Just _ | isInvalidDaml3Script fp -> fail "Daml3-script may only be used in LF 1.dev, and is unstable."
+            Just _ | isInvalidDaml3Script fp -> fail "Daml3-script may only be used in LF 2.dev, and is unstable."
```

Comment:
Any chance this could say `3.dev` if the goal is to make it stable with Daml 3? The integer police is unlikely to sue you for skipping over poor neglected 2, and longer-term it seems unlikely you'll run out of integers.

Reply:
The year is 4385, we're on LF (MAX_INT-1).dev, time is running thin...

Reply:
Seriously though, how many bits is this encoded in, in the tightest place? (Presumably a protobuf field.)

Reply:
Its hard to say, its passed around in so many places...
I believe most will have it as a string though

Reply:
For now, quite a lot of places (although less than before for 1.dev):
https://github.com/digital-asset/daml/pull/17348/files